src/cljs/rems/user_settings.cljs

```diff
@@ -80,10 +80,30 @@
                    (when first-time? ; to avoid an infinite loop if the settings fail to save (e.g. unsupported language in cookies)
                      [[:check-if-should-save-language!]]))})))
 
+(rf/reg-event-fx
+ :flash-message-error
+ (fn [_ user-settings-error]
+   ;; TODO: refactor this part
+   ;; this doesnt work
+   ;; (flash-message/default-error-handler :top "Fetch user settings")
+   ;; this works
+   (flash-message/show-default-error! :top (str "Fetch user settings")))
+ {})
+
+(rf/reg-event-fx
+ :loaded-user-settings-fail
+ (fn [{:keys [db]} [_ user-settings-error]]
+   (let [logged-in? (:user (:identity db))]
+     (if-not logged-in?
```

Comment:
Oh right, now I remember that discussion about 401. I think the point was that it would be nice to show an error when

- the user is not logged in
- *but* the settings API fails with some other code than 401

In other words, the error should _only be hidden if_:

- the user is not logged in
- and the error was 401